mozjs/js/rust/benches/htmlelement_expando_value_js.rs

```diff
@@ -0,0 +1,123 @@
+/* This Source Code Form is subject to the terms of the Mozilla Wlic
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#![feature(test)]
+#[macro_use]
+extern crate js;
+extern crate libc;
+extern crate test;
+
+use js::rust::{Runtime, SIMPLE_GLOBAL_CLASS};
+use js::rust;
+use js::debug::{val_to_str, puts};
+use js::jsapi::root::JS;
+use js::jsapi::root::{JS_NewGlobalObject, JS_InitClass, JSScript, JS_DefineFunction,};
+use js::jsapi::root::JS::CompartmentOptions;
+use js::jsapi::root::JS::OnNewGlobalHookOption;
+use js::jsapi::root::JS_ExecuteScript;
+use js::jsval::UndefinedValue;
+use js::magicdom::attr::ATTR_CLASS;
+use js::magicdom::attr::ATTR_PS_ARR;
+use js::magicdom::attr::Attr_constructor;
+use js::magicdom::element::ELEMENT_CLASS;
+use js::magicdom::element::ELEMENT_PS_ARR;
+use js::magicdom::element::ELEMENT_FN_ARR;
+use js::magicdom::element::Element_constructor;
+use js::magicdom::htmlelement::HTMLELEMENT_CLASS;
+use js::magicdom::htmlelement::HTMLELEMENT_PS_ARR;
+use js::magicdom::htmlelement::HtmlElement_constructor;
+use js::magicdom::node::NODE_CLASS;
+use js::magicdom::node::NODE_PS_ARR;
+use js::magicdom::node::Node_constructor;
+use test::Bencher;
+
+use std::ptr;
+
+#[bench]
+fn bench_htmlelement_setattributes_js(_b: &mut Bencher) {
```

Comment:
Do you intend for all of these to eventually become benches? If not, then they should be `#[test]`s under the `tests/` directory rather than `#[bench]`es in the `benches/` directory.